plugins/processors/t128_transform/t128_transform.go

```diff
@@ -12,13 +12,17 @@ import (
 )
```

Comment:
Are we including the "old" and "new" values in the data somewhere?

Reply:
This does not report the old value explicitly. I didn't think it makes sense in the context of the existing transforms and thought we'd concluded that we're just interested in suppressing repetitive output rather than identifying change edges per se.

Reply:
I had a different recollection - we had settled on including the old value (at least for the state case) as a tag so the user can decide whether they need it or not. For the ARP state change even for example, we should report the transition - the old state and the new state - that information is very much relevant

Reply:
Okay, will revisit

- [ ] report previous values